crates/ruff/src/rules/flake8_pie/rules.rs

```diff
@@ -136,6 +136,8 @@ pub fn no_unnecessary_pass(checker: &mut Checker, body: &[Stmt]) {
                 let mut diagnostic = Diagnostic::new(UnnecessaryPass, pass_stmt.range());
                 if checker.patch(diagnostic.kind.rule()) {
                     if let Some(index) = trailing_comment_start_offset(pass_stmt, checker.locator) {
+                        #[allow(deprecated)]
+                        #[allow(deprecated)]
```

Comment:
```suggestion
```

Reply:
Thanks! I'll go through the diff once more too and make sure I got all these — I started manually then switched to a regex replace and ended up with some duplicates and weird formatting :)